dp/drivers/ixgbe.c

```diff
@@ -1113,8 +1114,12 @@ int ixgbe_init(struct pci_dev *pci_dev, struct ix_rte_eth_dev **ethp)
 
 	if (!strcmp(driver->name, "rte_ixgbe_pmd"))
 		dev->dev_ops = &eth_dev_ops;
-	else if (!strcmp(driver->name, "rte_ixgbevf_pmd"))
+	else if (!strcmp(driver->name, "rte_ixgbevf_pmd")){
```

Comment:
if one of the if..else if..else clauses has braces, then all of them must have braces. See chapter 3 of https://www.kernel.org/doc/Documentation/CodingStyle